core/src/test/scala/neotypes/ConcurrentStreamingDriverSpec.scala

```diff
@@ -0,0 +1,35 @@
+package neotypes
+
+import neotypes.implicits.syntax.cypher._
+import org.scalatest.matchers.should.Matchers
+import scala.concurrent.ExecutionContext
+
+/** Base class for testing the concurrent use of a Driver. */
+final class ConcurrentStreamingDriverSpec[S[_], F[_]](
+  testkit: StreamTestkit[S, F]
+) extends StreamingDriverProvider(testkit) with BaseIntegrationSpec[F] with Matchers {
+  behavior of s"Concurrent use of Driver[${effectName}]"
```

Comment:
`StreamingDriver[${streamName}, ${effectName}]` instead of just `Driver[${effectName}]`